packages/@react-spectrum/actiongroup/docs/ActionGroup.mdx

```diff
@@ -0,0 +1,196 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/actiongroup';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {ActionGroup} from '@react-spectrum/actiongroup';
+import {ActionButton} from '@react-spectrum/button';
+import Brush from '@spectrum-icons/workflow/Brush';
+import Edit from '@spectrum-icons/workflow/Edit';
+import {Item} from '@react-stately/collections';
+import RegionSelect from '@spectrum-icons/workflow/RegionSelect';
+import Select from '@spectrum-icons/workflow/Select';
+```
+
+# ActionGroup
+
+<p>{docs.exports.ActionGroup.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ActionGroup']} />
+
+## Example
+
+```tsx example
+<ActionGroup>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+## Content
+
+ActionGroups accept as list of `children`, each with a `uniqueKey` prop.
+
+It is required that the `children` are Item components from the `@react-stately/collections` library.
+
+In order to identify children when a user takes an action on a item, each child needs to have a `uniqueKey` prop.
+See [Events](#events) for more information.
+
+## Value
+
+ActionGroups are uncontrolled by default. 
+
+Use `defaultSelectedKeys` to provide a default set of selected items. The value of the selected keys must match the `uniqueKey` prop of the items.
+
+```tsx example
+<ActionGroup defaultSelectedKeys={['delete']}>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+The `selectedKeys` prop can be used to make the selected state controlled.
+
+```tsx example
+<ActionGroup selectedKeys={['delete']}>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+### Accessibility
+
+If no visible label is provided (e.g. an icon only) an alternative text label must be provided to identify the control for accessibility.
+This should be added using the `textValue` prop to each Item.
```

Comment:
Does the ActionGroup itself need a visible label as well or an `aria-label`/`aria-labelledby` if one doesn't exist? Also IMO the first sentence should say something like "If no visible label is provided to the ActionButton(s)"  so  that it is clear that it is  talking about the Items/ActionButton visible label. Right now it reads as if providing `textValue` to each Item identifies the ActionGroup itself